ui/src/components/ResourceSelector.scss

```diff
@@ -0,0 +1,93 @@
+@import "../variables.scss";
+
+.resource-selector {
+  .MuiPaper-root {
+    background-color: #1a1a1a;
+    color: #e0e0e0;
+    border-radius: 8px;
+    border: 1px solid #333;
+    box-shadow: 0 0 10px rgba(0, 150, 255, 0.05);
+  }
+
+  .MuiDialogTitle-root {
+    text-align: center;
+    background-color: #222;
+    padding: 12px 20px;
+    font-size: 1.1rem;
+    font-weight: bold;
+    border-bottom: 1px solid #333;
+  }
+
+  .MuiDialogContent-root {
+    padding: 20px;
+  }
+
+  .resource-grid {
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    gap: 12px;
+    
+    @media (max-width: $sm-breakpoint) {
+      grid-template-columns: 1fr;
+    }
+  }
+
+  .resource-button {
+    height: 40px;
+    font-size: 0.9rem;
+    text-transform: none;
+    background-color: #2a2a2a;
+    border-radius: 4px;
+    transition: all 0.3s ease;
+    border: 1px solid #444;
+    overflow: hidden;
+    position: relative;
+
+    &:before {
+      content: '';
+      position: absolute;
+      top: -2px;
+      left: -2px;
+      right: -2px;
+      bottom: -2px;
+      background: linear-gradient(45deg, #00f2ff, #0066ff);
+      z-index: -1;
+      filter: blur(3px);
+      opacity: 0;
+      transition: opacity 0.3s ease;
+    }
+
+    &:hover {
+      transform: translateY(-1px);
+      &:before {
+        opacity: 0.3; // Further reduced opacity for even subtler glow
+      }
+    }
+
+    .wood { color: #009b00; } // Slightly less bright
+    .brick { color: #ff0000; }
+    .wheat { color: #ba9202; }
+    .sheep { color: #3eda58; }
+    .ore { color: #b0b0b0; }
+
+    .resource-name {
+      font-weight: bold;
+      text-shadow: none; // Removed text shadow
```

Comment:
Can we add a `margin: 4px;` here so that there is some space between the "+" and the resources? And also have the "+" be white? Trying to make it look something like this (and have the + more visible):
<img width="231" alt="Screenshot 2024-07-20 at 3 41 48 PM" src="https://github.com/user-attachments/assets/5855efe9-1ef4-4174-b9d9-60358a5f73c1">

Reply:
Good plan, if the bank only has one resource left, here's the x1 appearance as well
![image](https://github.com/user-attachments/assets/53d6f5bb-f8b0-4b7d-9c99-d2ce0fb96482)